packages/testing/src/injectable_test.ts

```diff
@@ -0,0 +1,41 @@
+import { SetMode } from '@seagull/mode'
+import { Container } from 'inversify'
+
+export abstract class InjectableTest {
+  abstract injectDiModules: any[]
+  abstract inject: any[]
+  injector = new Container()
+
```

Comment:
/** Hook that is to be executed before each and every single test without compromizing 
  *  the logic of the abstract class InjectionTest **/
beforeEach = noop